c/helpers/string.h

```diff
@@ -48,6 +48,15 @@ void remove_chars(char *str, int c, ...)
     }
 
     va_end(args);
+
+    // This is to confirm deleting the first vararg character
+    size_t j = 0;
+    for (size_t i = 0; str[i] != '\0'; ++i)
+    {
+        if (str[i] != c)
+            str[j++] = str[i];
+    }
+    str[j] = '\0';
```

Comment:
can we use [libunistring](https://conan.io/center/recipes/libunistring) instead?

Reply:
seems a really good candidate to standardize text processing as long as this is a build dependency.

I will check it out and see what can I do with it.

Reply:
Using this library has solved these inconsistencies between strings on different platforms